### PR TITLE
Feat/21 nc structured grid

### PR DESCRIPTION
**Pull request by AntoineSIMTEK**
Copy of muFFT's NCStructuredGrid NetCDFIO wrapper

### CODE REVIEW

**Comment by AntoineSIMTEK** [comment]
TODO: 
- [ ] add the license headers

File: test/IO/test_netcdf.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+"""
+
+Licence HEADER from muspectre
```
Comment (pastewka) [inline test/IO/test_netcdf.py:26]: Please remove the old muSpectre license header. Since I am the author, I am hereby relicensing this under MIT

**Review by pastewka — CHANGES_REQUESTED** [review]

File: test/IO/test_netcdf.py (hunk 1 above)
Comment (AntoineSIMTEK) [reply test/IO/test_netcdf.py:26]: Ok. Cab we do the same on the test ?

Comment (pastewka) [reply test/IO/test_netcdf.py:26]: Yes please

Comment (AntoineSIMTEK) [reply test/IO/test_netcdf.py:26]: Ah sorry this is the test. I also do that on the NetCDF file then

**Review by pastewka — APPROVED** [review]

**Pull request merged**